example_commands.md

```diff
@@ -11,7 +11,7 @@ docker run --rm --net cbio-net \
     -v "$PWD"/study-dir:/study:ro \
     -v "$HOME"/Desktop:/outdir \
     cbioportal-image \
-    metaImport.py -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html
+    sh -c 'exec metaImport.py --jar_path=/usr/local/tomcat/webapps/cbioportal/WEB-INF/lib/core-*.jar -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html'
```

Comment:
This command gets a bit complicated, as `cbioportalImporter.py` can only locate the jar file via the `$PORTAL_HOME` variable if it refers to a source folder with intermediate build files. I think it would be feasible to make the loading script check `$PORTAL_HOME/WEB-INF/lib/` as well, as this is where the jar file resides if `$PORTAL_HOME` refers to the deployed Java application.